spectrochempy/analysis/decomposition/iris.py

```diff
@@ -32,10 +32,15 @@
 from spectrochempy.utils.constants import EPSILON
 from spectrochempy.utils.decorators import signature_has_configurable_traits
 from spectrochempy.utils.docstrings import _docstring
-from spectrochempy.utils.optional import import_optional_dependency
 from spectrochempy.utils.traits import CoordType, NDDatasetType
 
-quadprog = import_optional_dependency("quadpog", errors="ignore")
+# note: import_optional_dependency does not work with quadprog
+# from spectrochempy.utils.optional import import_optional_dependency
+
+try:
+    import quadprog
+except ImportError:
+    pass
 
```

Comment:
May be it doesn't work because there is an typo in the "quadprog" name?

```quadprog = import_optional_dependency("quadpog", errors="ignore")```

should be 

```quadprog = import_optional_dependency("quadprog", errors="ignore")```   (quadpog-->quadprog)

Reply:
oh, yes. sorry for the supid error.